internal/action/bindings.go

```diff
@@ -251,6 +251,27 @@ func findEvent(k string) (Event, error) {
 	return event, nil
 }
 
+func compareEvents(a Event, b Event) bool {
+	switch aSequence := a.(type) {
```

Comment:
Hmm... My apologies, I've realized that the style I suggested makes it rather confusing in this case: it makes it non-obvious that the function behaves symmetrically with regard to both arguments.

Maybe something like:
```
func equalEvents(e1 Event, e2 Event) {
	seq1, ok1 := e1.(KeySequenceEvent)
	seq2, ok2 := e2.(KeySequenceEvent)
	if ok1 && ok2 {
		...
		return true
	}

	return e1 == e2
}
```

Reply:
So more or less as before...you're killing me. :D